Gemfile.lock

```diff
@@ -416,10 +416,13 @@ GEM
       rack (>= 1.1)
       rubocop (>= 1.33.0, < 2.0)
       rubocop-ast (>= 1.31.1, < 2.0)
-    rubocop-rspec (2.27.1)
+    rubocop-rspec (2.29.1)
       rubocop (~> 1.40)
       rubocop-capybara (~> 2.17)
       rubocop-factory_bot (~> 2.22)
+      rubocop-rspec_rails (~> 2.28)
+    rubocop-rspec_rails (2.28.2)
```

Comment:
You could consider adding `rubocop-rspec_rails` to `Gemfile` as it has been split out from `rubocop-rspec`. At the moment this isn't an issue but it will not be listed as a dependency of `rubocop-rspec` from version 3.0.